cli/rocks/rocks.go

```diff
@@ -44,6 +104,8 @@ func Exec(cmdCtx *cmdcontext.CmdCtx, cliOpts *config.CliOpts, args []string) err
 		cmd = "help"
 	}
 
+	cliOpts.Repo.Rocks = getRocksRepoPath(cliOpts.Repo.Rocks)
```

Comment:
`tt.yaml.default` example in repository does not have `tt.repo.rocks` configuration

Reply:
This is not an example config. This config is used for tt packages creation. It does not include rocks repository.